src/main/webapp/review.js

```diff
@@ -130,6 +128,7 @@ function storeComment(email) {
   const ratingListElement = document.getElementById('comment-list');
   ratingListElement.appendChild(createListElement(drinkObj));
   clearInput();
+  window.location.reload();//this will reload the page so the review section is shown as updated
```

Comment:
prefer adding a comment before the relevant line of code, and not at the end of the line, as it's easier to read that way.